study-builder/studies/testboston/snippets/picklist-question-yndk-prefernotanswer.conf

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
Actually this file can go into higher level snippets for use in other studies.  If needed we can add translation texts rather than substitution variables.

Reply:
I think it can stay here, since it uses the languages and translations specific to TestBoston.